codec-http2/src/main/java/io/netty/handler/codec/http2/Http2StreamChannelBootstrap.java

```diff
@@ -45,6 +45,9 @@
     private final Channel channel;
     private volatile ChannelHandler handler;
 
+    // Cache the ChannelHandlerContext to speed up open(...) operations.
+    private volatile ChannelHandlerContext multiplexCtx;
```

Comment:
Can you clarify if it is expected that the pipeline initialized with `Http2MultiplexCodec` is expected after the constructor (do we need/want this for 5.0 or is this just for compatibility)?

Reply:
I think it is just for compatibility for now. We can reconsider for Netty 5.